diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x autom4te.cache -x config.cache -x config.log -x config.status -x config.h -x config.h.in -x ABOUT-NLS -x Makefile.in.in -x Makevars.template -x '*SlackBuild*' -x '*.egg-info' -x '*.class' -x '*.pyc' -x '*.mo' -x '*.gmo' -x '*.orig' -x '*.rej' -x '*.spec' -x '*.temp' -x '*~' -x '*.stackdump' -x COPYING -x INSTALL -x compile -x config-ml.in -x config.guess -x config.sub -x depcomp -x elisp-comp -x install-sh -x ltmain.sh -x mdate-sh -x missing -x mkinstalldirs -x py-compile -x symlink-tree -x texinfo.tex -x ylwrap -x Makefile.in -x makefile.in -x configure -x omf.make -x xmldocs.make -x gnome-doc-utils.make -x gnome-doc-utils.m4 -x intltool.m4 -x intltool-extract -x intltool-extract.in -x intltool-merge -x intltool-merge.in -x intltool-update -x intltool-update.in origsrc/libogg-1.1.3/src/Makefile.am src/libogg-1.1.3/src/Makefile.am
--- origsrc/libogg-1.1.3/src/Makefile.am	2005-11-27 22:26:05.000000000 -0600
+++ src/libogg-1.1.3/src/Makefile.am	2007-07-29 20:06:27.625000000 -0500
@@ -17,7 +17,7 @@
 test_framing_SOURCES = framing.c
 test_framing_CFLAGS = -D_V_SELFTEST
 
-check: test_bitwise test_framing
+check: test_bitwise$(EXEEXT) test_framing$(EXEEXT)
 	./test_bitwise
 	./test_framing
 
